Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN go get golang.org/x/tools/cmd/goimports \
 	       github.com/go-ini/ini \
 	       github.com/jessevdk/go-flags \
 		   golang.org/x/net/context
+RUN go get ./...
```

Comment:
Yes, that might be easier indeed, or maybe we should start vendoring dependencies.

Reply:
We should use vendoring, there's an issue about that: #143.